### DISCUSSION TITLE
Is this possible to do?

### DISCUSSION THREAD

**Discussion by DaDaDaDara** [Budibase/budibase#10649] [Help] [answered]
Is there a way to set the value of a column to 0. I have a daily points column inside the budibase DB and i want to set it to 0 for all rows. I would prefer a button action, but any other way would be  great.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @DaDaDaDara,

I believe there is currently an active issue on this. Feel free to comment under here. This should hopefully bump the priority of this being fixed. https://github.com/Budibase/budibase/issues/10412

Currently you can use the `Update field value` action and provide a value/binding for it to update against. However this won't be display on the input field right now. (UI bug related to the issue linked above.) Even though you can't see it set as 0 when saved it should still update back to your DB as 0.

I hope this is helpful.
